src/services/reducers.js

```diff
@@ -51,32 +80,45 @@ const create = function(state = initialState.create, action) {
     case ACTION_TYPES.CREATE_PHOTO_ADD_MEMBER:
       return {
         ...state,
-        [STORE_TYPES.STATE.CREATE.MEMBER]: [
+        [STORE_TYPES.STATE.CREATE.MEMBER]: {
           ...state[STORE_TYPES.STATE.CREATE.MEMBER],
-          { photo_member: action.data }
-        ]
+          [action.data]: {
+            photoTypeNumber: {
+              yori: 0,
+              chu: 0,
+              hiki: 0,
+              suwari: 0
```

Comment:
I think these types can be extracted as constants. But it is OK to fix them in next PR.